web/source/osk/visualKeyboard.ts

```diff
@@ -523,7 +545,12 @@ namespace com.keyman.osk {
       '*LAltShift*':      0x67,
       '*RAltShift*':      0x68,
       '*LCtrlShift*':     0x69,
-      '*RCtrlShift*':     0x70
+      '*RCtrlShift*':     0x70,
+      '*RtlEnter*':       0x71,
+      '*RtlBkSp*':         0x72,
+      '*ShiftLock*':       0x73,
+      '*ShiftedLock*':     0x74,
+      '*ZWNJ*':           0x75, // for iOS version, 0x76 for Android version.
```

Comment:
The relevant part for use within Developer.

Reply:
Note that code elsewhere within this set of changes dynamically selects the best PUA for the platform.

... though, we may want some discussion on which PUA actually is "best" for each platform.

Reply:
I've added #3863 to track this change in Developer.

Reply:
Unsure if we should use `*RTLEnter*` instead of `*RtlEnter*` (and bksp of course)? Given `ZWNJ` is all caps, perhaps we should do the same for `RTL`?

```suggestion
      '*RCtrlShift*':     0x70,
      '*RtlEnter*':       0x71,
      '*RtlBkSp*':        0x72,
      '*ShiftLock*':      0x73,
      '*ShiftedLock*':    0x74,
      '*ZWNJiOS*':        0x75, // *ZWNJ* will select iOS/Android version automatically
      '*ZWNJAndroid*':    0x76, // *ZWNJ* will select iOS/Android version automatically
```

Reply:
> Unsure if we should use `*RTLEnter*` instead of `*RtlEnter*` (and bksp of course)? Given `ZWNJ` is all caps, perhaps we should do the same for `RTL`?

Honestly... I was kind of back and forth on the `RTL` vs `Rtl` bit.  Given the other codes in the file (like `Ctrl` instead of `CTRL`), I decided to go with the initial-cased variant.

I'd normally prefer go with the same (initial-cased) for `ZWNJ`... but Apple-casing causes a nasty visual conflict.  `ZwnjiOS` isn't very clear, and `ZwnjIos` has its own issues.  So, I felt `ZWNJiOS` was the clearest option there.

Reply:
The one 'exception' to that "other codes" bit, IMO, is `*BkSp*`... and even then, it's `*BkSp*`, not `*BKSP*`.

Reply:
`CTRL` is not an acronym, though, unlike `RTL` and `ZWNJ`, so that's why I was going with that approach. Admittedly, the legacy `BkSp` is pretty ugly!

I think I'd like to go with `RTLEnter`, `RTLBkSp`, and `ZWNJ` (and its variants).

For the future, we should make these case-insensitive. The Keyman keyboard language identifiers are case insensitive and it doesn't make sense to make these case-sensitive. But that's probably not a job for this PR.

Reply:
> For the future, we should make these case-insensitive. The Keyman keyboard language identifiers are case insensitive and it doesn't make sense to make these case-sensitive. But that's probably not a job for this PR.

And ... we can't: we have `*ABC*` and `*abc*` already defined. Boowah.